src/modules/kpi-reporting/lib/mappers.js

```diff
@@ -43,7 +43,7 @@ const mapLicenceNamesData = (data) => {
       acc.monthly.push(
         {
           ...row,
-          month: months[row.month],
+          month: months[row.month - 1],
```

Comment:
Again, not a problem, just an advisory: It might be worth adding an inline comment here to explain the -1 (as I'm not sure why it's -1, but I'm sure there's a very good reason!) :)

Reply:
good shout I added a comment -- it's because of the array starting with index 0.